x-pack/plugins/cases/public/containers/api.test.tsx

```diff
@@ -125,6 +126,34 @@ describe('Case Configuration API', () => {
     });
   });
 
+  describe('resolveCase', () => {
+    const targetAliasId = '12345';
+    const basicResolveCase = {
+      outcome: 'aliasMatch',
+      case: basicCaseSnake,
+    };
+    const caseId = basicCase.id;
+
+    beforeEach(() => {
+      fetchMock.mockClear();
+      fetchMock.mockResolvedValue({ ...basicResolveCase, target_alias_id: targetAliasId });
+    });
+
+    test('check url, method, signal', async () => {
+      await resolveCase(caseId, true, abortCtrl.signal);
+      expect(fetchMock).toHaveBeenCalledWith(`${CASES_URL}/${caseId}/resolve`, {
+        method: 'GET',
+        query: { includeComments: true },
+        signal: abortCtrl.signal,
+      });
+    });
+
+    test('happy path', async () => {
```

Comment:
I would say we move away from 'happy path' as a naming convention as a new engineer to the team may not know what 'happy path' really means. Or if we want to keep the 'happy path' naming convention, then maybe make the 'happy path' it's own describe block...so something along the lines of:

```
describe('happy path', () => {
	`test('should return an response with case, targetAliasId, and expected case attributes set')`
})
```